<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBook_X1" failures="1" tests="3" name="Test" time="8.399" errors="0" timestamp="2025-04-04T21:20:47 IST">
  <testcase classname="com.amazon.tests.AddToCartTest" name="addProductToCart" time="0.009"/>
  <testcase classname="tests.LoginTest" name="loginTest" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: src\test\resources\testdata.xlsx (The system cannot find the path specified)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: src\test\resources\testdata.xlsx (The system cannot find the path specified)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.io.FileNotFoundException: src\test\resources\testdata.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
at utils.ExcelUtil.getExcelData(ExcelUtil.java:17)
at dataprovider.DataProviderClass.getTestData(DataProviderClass.java:9)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="tests.SearchTest" name="searchTest" time="0.002"/>
</testsuite> <!-- Test -->
